### ISSUE TITLE
Weird "Show More" Scrolling Behavior

### ISSUE THREAD

**Issue by IRlyLikeBac0n** [dangeredwolf/ModernDeck#175]
Sometimes when I accumulate a bunch of new unread tweets in a column, I click the "Show More" button and am immediately scrolled to the top (newest) of the unread tweets. This behavior may be intentional, but when I then go to scroll down (I prefer catching up to tweets from the bottom up), the newly-loaded tweets "reset", and I am taken back to the starting position. I then have to click "Show More" again and start over.

I'm not sure if this is caused by there being too many unread tweets to load at once, or some scrolling bug, but I figured I'd create the issue to see if anyone else has encountered it before.

**Comment by dangeredwolf** [comment]
Yes, this is a behavior of TweetDeck. It might be difficult, but not impossible to implement such a thing. I'll keep this issue open and see if I can make it behave more properly.
